README.md

```diff
@@ -12,28 +12,22 @@ Helpers for sending [Datadog custom metrics](https://docs.datadoghq.com/develope
 yarn add seek-datadog-custom-metrics
 ```
 
-## Tagging convention
+## Naming convention
 
-All custom metrics are prefixed by `AppConfig.name`.
-Two global tags are also added to every custom metric:
-
-- `AppConfig.environment` becomes `env:${value}`
-- `AppConfig.version` becomes `version:${value}`
-
-These tags are consistent with tags sent by [Gantry](https://github.com/SEEK-Jobs/gantry) via Datadog's AWS integration.
+All custom metrics are prefixed by `{config.name}.`.
```

Comment:
I'm lazily proposing we keep this. Other options may include automatic prefixing with `DD_SERVICE`.

Reply:
Would there be value in DD_SERVICE or SERVICE? 🤔 
Might make logs 🤝 metrics a nicer experience in Datadog for those not using OTel?